### DISCUSSION TITLE
System Informer 3.0.7435 downloaded 500MB of data on Windows 11

### DISCUSSION THREAD

**Discussion by poqdavid** [winsiderss/systeminformer#1979] [Questions / Answers] [answered]
I have been using System Informer for a while and I recently updated it to `Version: 3.0.7435 (0524888)`. After the update, I opened the properties of a process and noticed that System Informer had downloaded about 500MB of data in the background.

I checked the network usage and noticed that the usage was from System Informer's process, but I could not identify the source or destination of the data. I did not receive any notification or prompt from System Informer about this data transfer.

So, I am wondering what System Informer downloaded without my knowledge. Is this a normal behavior of System Informer?
How can I find out what data was downloaded and where it was stored?

I would appreciate any help or advice. Thank you.

**Comment by Biswa96** [comment, marked as answer]
Did you download PDB files for system shared libraries and executables? The threads tab of a process window downloads PDB automatically.

**Reply by poqdavid** [reply]
Oh, I see yes, I did open the threads tab tyvm :)
I wish System Informer had a notification or popup about that at least.

**Reply by dmex** [reply]
When you've installed the Debugging Tools for Windows via the Windows SDK/WDK then it'll download Microsoft's debugging symbols into C:\Symbols and resolve addresses, thread stacks and object names.

> I wish System Informer had a notification or popup about that at least.

There's progress bars and status text on each window showing the activity.

**Reply by poqdavid** [reply]
> When you've installed the Debugging Tools for Windows via the Windows SDK/WDK then it'll download Microsoft's debugging symbols into C:\Symbols and resolve addresses, thread stacks and object names.
> 
> > I wish System Informer had a notification or popup about that at least.
> 
> There's progress bars and status text on each window showing the activity.

Oh, tyvm for the information :)
